azure-mgmt-graph-rbac/src/test/java/com/azure/management/graphrbac/GraphRbacManagementTest.java

```diff
@@ -24,7 +26,14 @@
 
     @Override
     protected void initializeClients(RestClient restClient, String defaultSubscription, String domain) {
-        graphRbacManager = GraphRbacManager.authenticate(restClient, domain);
+        RestClient graphRestClient;
+        if (isPlaybackMode()) {
+            ApplicationTokenCredential credentials = new AzureTestCredential(playbackUri.replace("http", "https"), ZERO_TENANT, true);
```

Comment:
`http://` to `https://` might be better against accidental match?

Reply:
I'm planning to refactor the constructor of GraphRbacManager/MsiManager. If we apply same constructor like that in KeyVaultManager to pass parameters by `RestClient`, `tenantId`, `subscriptionId` instead of only `RestClient` and `tenantId`. After that, we can remove this block